common/djangoapps/student/models.py

```diff
@@ -1147,7 +1147,8 @@ def course_id(self, value):
     MODE_CACHE_NAMESPACE = u'CourseEnrollment.mode_and_active'
 
     class Meta(object):
-        unique_together = (('user', 'course'),)
+        unique_together = (('user', 'course'),
+                           ('user', 'created'))
```

Comment:
1. The user might have multiple enrollments created for them at the same time, so this shouldn't be a unique constraint. Please use [`index_together`](https://docs.djangoproject.com/en/1.11/ref/models/options/#index-together) instead.
2. We want to index on descending time created, so please use `('user', '-created')`.

Reply:
@ormsbee while using `-created` I was getting following error `index_together refers to the non-existent field '-created'`. So that's why I had used `created`